src/doc/rustc-dev-guide/src/traits/implied-bounds.md

```diff
@@ -55,7 +55,7 @@ The assumed outlives constraints for implicit bounds are computed using the
 MIR borrowck adds the outlives constraints for both the normalized and unnormalized types,
 lexical region resolution [only uses the unnormalized types][notnorm].
 
-[`fn OutlivesEnvironment::with_bounds`]: https://github.com/rust-lang/rust/blob/5b8bc568d28b2e922290c9a966b3231d0ce9398b/compiler/rustc_infer/src/infer/outlives/env.rs#L90-L97
+[`fn OutlivesEnvironment::new`]: TODO
```

Comment:
This is an interesting situation where I don't yet have a commit hash to put here until this is ready to land :>

Reply:
You could just pust `master` instead of a commit hash for now

Reply:
why not https://github.com/rust-lang/rust/blob/485c6f4ba46339feb7e9256f338a1a3208dcb7e0/compiler/rustc_infer/src/infer/outlives/env.rs#L90-L97 ?